src/views/home/Discussion.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div>
     <div class="discussion">
+      <div class="py-2 new">
+        <router-link
+          class="link"
+          to="/communities/series-e72204a6/discussion/new"
```

Comment:
```suggestion
          to=`/communities/${this.$route.params.comminitySlug}/discussion/new`
```
burada artık url'den gelen slug değeri ile dinamik bir şekilde yönlendirme yapabiliriz